### PR TITLE
cleanup: Remove unused tekton and virtioImage code

### PR DESCRIPTION
**Pull request by akrejcir**
**What this PR does / why we need it**:
This removes minor usages of tekton from these places:
- CSV generator
- Environment variables used by operator
- SSP CR example
- Default update function for reconcile code
- SSP CR created by functional tests

Removes unused `virtioImage` code.

**Release note**:
```release-note
None
```

### CODE REVIEW

File: internal/common/environment.go [hunk 1]
```diff
@@ -18,30 +18,16 @@ import (
...
 const (
-	OperatorVersionKey          = "OPERATOR_VERSION"
-	TemplateValidatorImageKey   = "VALIDATOR_IMAGE"
-	TektonTasksImageKey         = "TEKTON_TASKS_IMAGE"
-	TektonTasksDiskVirtImageKey = "TEKTON_TASKS_DISK_VIRT_IMAGE"
-	VirtioImageKey              = "VIRTIO_IMG"
-	VmConsoleProxyImageKey      = "VM_CONSOLE_PROXY_IMAGE"
+	OperatorVersionKey        = "OPERATOR_VERSION"
+	TemplateValidatorImageKey = "VALIDATOR_IMAGE"
+	VirtioImageKey            = "VIRTIO_IMG"
```
Comment (ksimon1) [inline internal/common/environment.go:23]: This can be deleted too

**Comment by akrejcir** [comment]
HCO does not use the parameters.

/unhold

File: internal/common/environment.go (hunk 1 above)
Comment (akrejcir) [reply internal/common/environment.go]: done, in a new commit

File: internal/common/environment.go [hunk 2]
```diff
@@ -18,30 +18,16 @@ import (
...
+	VmConsoleProxyImageKey    = "VM_CONSOLE_PROXY_IMAGE"
 
-	DefaultTektonTasksIMG         = "quay.io/kubevirt/tekton-tasks:" + TektonTasksVersion
-	DeafultTektonTasksDiskVirtIMG = "quay.io/kubevirt/tekton-tasks-disk-virt:" + TektonTasksVersion
-	DefaultVirtioIMG              = "quay.io/kubevirt/virtio-container-disk:v1.2.0"
+	DefaultVirtioIMG = "quay.io/kubevirt/virtio-container-disk:v1.2.0"
```
[the remark this answers is not in the document]
Comment (akrejcir) [reply internal/common/environment.go]: done

**Comment by 0xFelix** [comment]
What about `docs/configuration.md`, `docs/development.md` and https://github.com/kubevirt/ssp-operator/blob/b717b6851da2e3d8bebf983248f378ea8cbdb58b/internal/common/version.go#L4?

**Comment by akrejcir** [comment]
> What about `docs/configuration.md`, `docs/development.md` and
> 
> https://github.com/kubevirt/ssp-operator/blob/b717b6851da2e3d8bebf983248f378ea8cbdb58b/internal/common/version.go#L4
> 
> ?

The documentation is removed in https://github.com/kubevirt/ssp-operator/pull/945

I missed the `version.go` file, thanks for noticing.

**Review by 0xFelix — COMMENTED** [review]
Thanks!

/approve
/retest

**Pull request merged**